### ISSUE TITLE
Re-starting the project

### ISSUE THREAD

**Issue by gabrielfgularte** [gabrielfgularte/QRSocial#5]
Hi guys, I came up with an idea to build this in a simple way.

We build an webapp (in Node, Python, Ruby, whatever) with 3 routes/views/pages.

1. **Home** to explain the project. It will be static and can be a simple GitHub Page.

2. **Create QRCode** to bloggers enter their social accounts. We provide - to start - Facebook, Twitter, Youtube and Instagram. So we render a list with 4 items (facebook.com/, instagram.com/, etc) and a form for each item, so the blogger will fill the form only with its username for each social media. After blogger fills and submit, we create an url with params that represent the usernames (something like: `qrsocial.com/follow?fb=blogger_username&twitter=blogger_username&youtube=...`) and with that URL we create the QRCode. Anyone that reads that QR will be taken to our next view/page/route: **Read QRCode**.

3. **Read QRCode** will read the URL params and search for our social media keywords (fb, twitter, etc). If we find any social media keywords, we get its value and generate 4 urls that we're gonna render in this page as links - to start* - so the reader can click and follow the blogger.

4. My english is terrible. And I have no time to throw all in translator and adjust. Sorry for that.

5. What do you think?

* We can add connection with APIs after, as well as toggle buttons and much more. But, I think links will solve the problem and its a good start point.

@barbier
@bernardocs

**Comment by barbier** [comment]
I liked the architecture!
Since we are 3 JS developers, I think it's going to be better if we do in Node, but I don't know how to build a QR Code.... Never found something useful.....
For the Frontend, can we keep the simple and do it with jQuery?

**Comment by gabrielfgularte** [comment]
Yeah, sure! Let's keep it simple and dev in Node. I'm gonna search for something about QRCode. Could you do the same?

**Comment by gabrielfgularte** [comment]
@barbier I've found this: https://kazuhikoarase.github.io/qrcode-generator/ and it's npm wrapper: https://www.npmjs.com/package/qrcode-js.

I'm gonna build the web application first. It will be my first real world project using Node, and the last time I've played with Node/Express was about a year ago.

I'm thinking about use Heroku to host the application. And we have to buy some domain. So:

1. What do you think about Heroku? Have you guys worked with it?
2. I'm gonna buy the domain on GoDaddy. Any suggests? Other services?
3. I'm thinking on qrsocial.com. Good?

**Comment by barbier** [comment]
I liked this library!
Seems easy to work with and do the job.

1. I have never worked with Heroku before, so I don't know if it's easier and cheaper than Amazon or Azure.
2. I don't know any other.... But I have some domains on it!
3. QRSocial.com seems fine!
4. Should we try to put an ad on it so the project will sustain itself on a long run?

**Comment by gabrielfgularte** [comment]
4. Fill with adds for the start to do some money.

**Comment by gabrielfgularte** [comment]
The Create QRCode page looks like this in my head

![page 1](https://cloud.githubusercontent.com/assets/1307137/18104001/7416653a-6ecf-11e6-9d0b-e37753523dab.png)

**Comment by barbier** [comment]
Awesome!
Let's do it!

**Comment by gabrielfgularte** [comment]
Now we have to decide who gonna do what, right? Let me start the project e push an initial version. Then, we can proceed with more issues and doubts about the project, ok?
